### PR TITLE
Make settings consistent over different web repos

### PR DESCRIPTION
**Pull request by SaptakS**
- [x] Remove the HSTS settings, other apps leave this to the TLS termination level
- [x] base.py should default DEBUG to False so that we can reason about it in the same way as other apps
- [x] Stub out CSP variables in the same manner as SDO so that we can use them later
- [x] Simply the process of adding whitenoise to middleware (follow SDO and PFT, instead of complex calculating-an-index logic)
- [x] Instead of DJANGO_LOG_CONSOLE and DJANGO_LOG_PATH, just log to console if a log path is not set. Don't require DJANGO_LOG_JSON, default it appropriately.
- [x] Treat env vars that should be "booleans" the same everywhere.

### CODE REVIEW

**Comment by maeve-fpf** [comment]
Thanks @SaptakS! I've added:

- Simplified JSON logging to check that box
- a python-json-logging upgrade for the above
- Other misc dependency updates (base images also)
- Database environment config

File: securethenews/settings/base.py [hunk 1]
```diff
@@ -295,3 +297,87 @@
     'DJANGO_CSP_REPORT_URI',
     'https://freedomofpress.report-uri.com/r/d/csp/enforce'
 )
+
+
+# Logging
+#
+# Logs are now always JSON. Normally, they go to stdout. To override this for
+# development or legacy deploys, set DJANGO_LOG_DIR in the environment.
```
Comment (chigby) [inline securethenews/settings/base.py:305]: Here we say `DJANGO_LOG_DIR` but below we are checking `DJANGO_LOG_PATH` in the environment.

**Review by chigby — APPROVED** [review]
These changes look good to me, glad to see that we're actively working on updating this code and keeping all of our sites consistent with each other. 

One question I have: is there something that, for my dev environment, I can adjust somewhere to change to less verbose plaintext style logging? I definitely see the advantages of JSON it can make the log console somewhat harder to read.

I made one small note about a comment that seems to disagree with the code. Feel free to change and merge as you see fit.

File: securethenews/settings/base.py (hunk 1 above)
Comment (maeve-fpf) [reply securethenews/settings/base.py:305]: Thanks, updated the code to match the comment (here and in the other repos).

**Comment by maeve-fpf** [comment]
Thank you! It should not be too complicated to use the plain text formatter in dev again (I didn't want to add more complexity with other things to review), but I'd like to get these deployed first if possible.

**Comment by chigby** [comment]
Got it. I'll merge when CI finishes.

**Pull request merged**